odoo/addons/base/static/tests/base_settings_tests.js

```diff
@@ -424,5 +431,58 @@ QUnit.module('base_settings_tests', {
 
         actionManager.destroy();
     });
+
+    QUnit.test('settings can contain one 2 many fields', async function (assert) {
+
+        assert.expect(2);
+
+        const form = await createView({
+            View: BaseSettingsView,
+            model: 'res.config.settings',
+            data: this.data,
+            arch: `
+                   <form string="Settings" class="oe_form_configuration o_base_settings">
+                       <header>
+                           <button string="Save" type="object" name="execute" class="oe_highlight" />
+                           <button string="Discard" type="object" name="cancel" special="cancel" />
+                       </header>
+                       <div class="o_setting_container">
+                           <div class="settings_tab"/>
+                           <div class="settings">
+                               <div class="notFound o_hidden">No Record Found</div>
+                               <div class="app_settings_block" string="Base Setting" data-key="base-setting">
+                                   <field name="tasks">
+                                       <tree>
+                                            <field name="display_name"/>
+                                       </tree>
+                                   </field>
+                               </div>
+                           </div>
+                       </div>
+                   </form>`,
+            archs: {
+                'task,false,form': '<form><field name="display_name"/></form>',
```

Comment:
you can inline this one as well (alonside the `<tree>` one)

Reply:
You mean doing this: ?
```js
<field name="tasks">
    <tree><field name="display_name"/></tree>
</field>
```

Reply:
I mean, in the main arch:
```xml
                                   <field name="tasks">
                                       <tree><field name="display_name"/></tree>
                                       <form><field name="display_name"/></form>
                                   </field>
```
Each of them on a single line or not, I don't really care. But that way, no need to define the `archs` keys, as all archs are defined in the main one.

Reply:
Oh ok I didn't know that was a thing !